server/src/main/java/com/testsigma/service/TestsigmaLabDriverSettingsService.java

```diff
@@ -119,7 +119,7 @@ public void setWebCapabilities(TestDevice testDevice,
       Float.parseFloat(platformBrowserVersion.getDisplayVersion()) > 12) {
```

Comment:
@vikramvr20 do we need this one, I guess LT have made fixes above issue.. we can avoid passing selenium_version as capability